docs/design/endpoints.md

```diff
@@ -117,41 +296,53 @@ public open class EndpointProviderGenerator(
     public fun render()
 
     /**
-     * Resolve a standard library function referenced in a ruleset.
+     * Render a standard library function referenced in a rule set.
      */
-    public open fun resolveFunction(func: String)
+    public open fun renderFunction(func: String, args: List<FunctionArg>)
```

Comment:
If these are standard library functions then I would think `resolveFunction(name: String): Symbol` would be appropriate right? The generator obviously knows the parameters already since they are a function argument so simply resolving to a symbol would be enough.

e.g.

```kotlin
// hypothetical codegen path in render()

val xyzStdlibFn = resolveFunction("xyz")
writer.write("val xyzResult = #T(param1, param2)", xyzStdlibFn)